presto-verifier/src/main/java/com/facebook/presto/verifier/rewrite/QueryRewriter.java

```diff
@@ -166,11 +173,43 @@ public QueryObjectBundle rewriteQuery(@Language("SQL") String query, ClusterType
                     ImmutableList.of(new DropTable(temporaryTableName, true)),
                     clusterType);
         }
+        if (statement instanceof CreateView) {
+            CreateView createView = (CreateView) statement;
+            QualifiedName temporaryViewName = generateTemporaryName(Optional.empty(), prefix);
+            ImmutableList.Builder<Statement> setupQueries = ImmutableList.builder();
+
+            // Check to see if there is an existing view with the specified view name.
+            // If view exists, create a temporary view that are has the same definition as the existing view.
+            // Otherwise, do not pre-create temporary view.
+            try {
+                CreateView createExistingView = (CreateView) sqlParser.createStatement(
+                        getOnlyElement(prestoAction.execute(new ShowCreate(VIEW, createView.getName()), REWRITE, SHOW_CREATE_VIEW_CONVERTER).getResults()),
```

Comment:
Should we extract this into a method?

Reply:
This method would only be used once if we extract. I'm changing this to 2 statements - that might help readability.